### ISSUE TITLE
The blocks downloaded indicator keeps restarting

### ISSUE THREAD

**Issue by grantspilsbury** [ethereum/mist#2811]
I'm new to everything block chain. 
I've just downloaded Mist for the first time. I entered a password and have chosen testnet. Blocks have started to download but when it reaches around 20% / 250,000 blocks downloaded it shows that it starts again from 0% (or very low %). It has been running / downloading for about 5 hours now and keeps going up to 200,000 blocks downloaded and then back down to 0%. 

Is this expected behaviour?

My internet connection seems to be stable and I'm using a MacBook Pro 15inch with 16gig ram and 1Tb hard drive.

**Comment by sathishvj** [comment]
No, I don't think that is expected behavior.  Make sure you backup your keystore files always!  After that you can delete the chaindata and start the process again.
